packages/core/parcel/src/cli.js

```diff
@@ -101,10 +104,12 @@ var hmrOptions = {
 
 function applyOptions(cmd, options) {
   for (let opt in options) {
-    cmd.option(
-      opt,
-      ...(Array.isArray(options[opt]) ? options[opt] : [options[opt]]),
-    );
+    const option = options[opt];
+    if (option.flags) {
```

Comment:
Does this check the same as `option instanceof commander.Option` or are there actually `Option` objects with no `flags` and we want to discern them?

Reply:
Did you undo the fix for this on purpose?

Reply:
Looks like a rebase fail on my part